roles/openshift_health_checker/openshift_checks/logging/elasticsearch.py

```diff
@@ -62,7 +62,9 @@ def running_elasticsearch_pods(self, es_pods):
 
     @staticmethod
     def _build_es_curl_cmd(pod_name, url):
-        base = "exec {name} -- curl -s --cert {base}cert --key {base}key --cacert {base}ca -XGET '{url}'"
+        base = "exec {name} -c elasticsearch " \
+             "-- curl -s --cert {base}cert --key {base}key " \
```

Comment:
```suggestion
               "-- curl -s --cert {base}cert --key {base}key " \
```
to fix `E126 continuation line over-indented for hanging indent`